server/middleware.json

```diff
@@ -10,6 +10,9 @@
         "credentials": true,
         "maxAge": 86400
       }
+    },
+    "cookie-parser": {
+      "params": "DecodrRing"
```

Comment:
I don't know what this means or why we need it. But without it it doesn't work :\